### ISSUE TITLE
Getting different test results on the KITTI

### ISSUE THREAD

**Issue by ZM-Zhou** [prstrive/EPCDepth#8]
Hi, first of all, thanks for your excellent work!  When I tried to reproduce your results on the KITTI test set with your code and pretrained weights, I got different results from those reported in this repository. Specifically, I tested `model50` with:
`python main.py 
    --val --data_path <kitti path> --resume <model path>/model50.tar 
    --use_full_scale --post_process --output_scale 0 --disps_path <disparity save path>
    --num_layer 50 --batch_size 4`
And the results are:
From| Abs Rel | Sq Rel | RMSE | δ < 1.25
-- |-- | -- | -- | --
This Repository| 0.091 | 0.646 | 4.207 | 0.901
My Reproduction|  0.096|  0.669|   4.254|    0.888

It is noticed that the extension name of the images in my KITTI dataset is `.png`, and you mentioned that  'Our pre-trained model is trained in jpg, and the test performance on png will slightly decrease.'. Are the differences just caused by the extension name of the images? Or do I misunderstand other things?

**Comment by prstrive** [comment]
I think so.
